Playbooks/playbook-Accessdata__Dump_memory_for_malicious_process.yml

```diff
@@ -0,0 +1,446 @@
+id: Accessdata_Dump_memory_for_malicious_process
+version: -1
+fromversion: 4.1.0
+name: 'Accessdata: Dump memory for malicious process'
+description: This playbook dumps memory  if given process is running on legacy AD
+  agent
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 8ad54dd6-b579-41ef-81da-1423e2aa8f4f
+    type: start
+    task:
+      id: 8ad54dd6-b579-41ef-81da-1423e2aa8f4f
+      version: -1
+      name: ""
+      description: Playbook entry point
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 10
+        }
+      }
+    note: false
+    timertriggers: []
+  "1":
+    id: "1"
+    taskid: 6c822883-2638-48b1-8c41-c09e0eb4a633
+    type: regular
+    task:
+      id: 6c822883-2638-48b1-8c41-c09e0eb4a633
+      version: -1
+      name: Get process list from legacy agent
+      description: Call command to get process list from legacy agent
+      script: '|||accessdata-legacyagent-get-processlist'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "15"
+    scriptarguments:
+      caseid:
+        simple: ${inputs.caseid}
+      target_ip:
+        simple: ${inputs.target_ip}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 290
+        }
+      }
+    note: false
+    timertriggers: []
+  "7":
+    id: "7"
+    taskid: 15ba3de3-acd0-4849-8f4c-6e9c37a559d2
+    type: title
+    task:
+      id: 15ba3de3-acd0-4849-8f4c-6e9c37a559d2
+      version: -1
+      name: Done
+      description: Playbook exit point
+      type: title
+      iscommand: false
+      brand: ""
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1715
+        }
+      }
+    note: false
+    timertriggers: []
+  "8":
+    id: "8"
+    taskid: f496a274-70f8-4bb7-85dc-6f0973b5e0fc
+    type: regular
+    task:
+      id: f496a274-70f8-4bb7-85dc-6f0973b5e0fc
+      version: -1
+      name: DeleteContext
+      description: Clear current context
```

Comment:
we dont use DeleteContext in playbooks, only for test playbook purposes. please delete this step, unless it is necessary